source/MaterialXRuntime/RtFileIo.h

```diff
@@ -19,18 +19,61 @@
 namespace MaterialX
 {
 
-/// API for read and write of data from MaterialXCore documents
-/// to MaterialXRuntime stages.
-class RtFileIo : public RtApiBase
+/// @class RtReadOptions
```

Comment:
Clean up so there is 1 read and 1 write options structure. Also hides the XML options which are updated as required internally.